packages/vc-status-rl-2020/src/typings.d.ts

```diff
@@ -1 +1,2 @@
 declare module 'jsonld';
+declare module '@transmute/vc.js';
```

Comment:
no, we don't want to destroy types like this.

Reply:
I will look at removing this. if I don't have this type here, it breaks inside the github action with this error: https://github.com/transmute-industries/verifiable-data/runs/5059846433?check_suite_focus=true#step:4:713

Reply:
Removing this!

Reply:
So I have removed this and encountered this error in CI action again: https://github.com/transmute-industries/verifiable-data/runs/5069492667?check_suite_focus=true#step:4:682 any ideas on what issue is? `npm i` works without issue locally.

Reply:
You are in a lerna repo, you need to lerna bootstrap.